src/lib.rs

```diff
@@ -264,6 +273,17 @@ fn try_convert_duration_to_go_type(from: Duration) -> Result<i64, StartError> {
     i64::try_from(from.as_nanos()).map_err(|_| StartError::DurationIsTooLong(from))
 }
 
+#[allow(clippy::missing_panics_doc)]
+// ^^^ We know it's safe to unwrap because "" does not contain any null bytes
```

Comment:
if it's safe to unwrap, could unwrap inside this function and return a `CString` instead of a `Result`?

Reply:
It took me a bit to understand that this comment refers to line 283, I thought this comment meant that it's safe to call `access_token_as_cstring(t).unwrap()`. Could we make that more clear?

Reply:
Great feedback 👍🏻

Reply:
See 002348d, is it more clear now?